MonoGame.Framework/Game.cs

```diff
@@ -19,6 +19,10 @@ namespace Microsoft.Xna.Framework
 {
     public class Game : IDisposable
     {
+#if DESKTOPGL
+        public static bool Resizable = false;

```

Comment:
> I'm taking about OpenTK ... not OpenTK with SDL. Did/does OpenTK support changing the window to resizable after creation?

You don't get how opentk functions, you can't ask "does OpenTK support this" since it depends on the backend it runs with:
1) OpenTK has 2 backends for each platform, native and SDL
2) During initialization it will pick one (this can be forced within code)
3) For resizable, If it picked SDL it won't support it, if it picked native it will support it

All an all OpenTK can't help SDL with this. 

We can't use OpenTK to make SDL window resizable.

PS. SDL 1.2 did support changing resizable parameter after window creation.